<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Microsoft Tai Le</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QRadioButton" name="loadCustom">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>270</y>
      <width>81</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Custom</string>
    </property>
   </widget>
   <widget class="QLabel" name="threadText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>330</y>
      <width>55</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Threads:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="loadMedium">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>212</y>
      <width>84</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Medium</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="threadSelect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>340</y>
      <width>56</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QLabel" name="loadText">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>300</y>
      <width>51</width>
      <height>50</height>
     </rect>
    </property>
    <property name="text">
     <string>Load:</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="loadHigh">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>240</y>
      <width>61</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>High</string>
    </property>
   </widget>
   <widget class="QPushButton" name="beginTest">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>460</y>
      <width>80</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Begin</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="loadLow">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>56</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Low</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="loadSelect">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>56</width>
      <height>30</height>
     </rect>
    </property>
    <property name="maximum">
     <number>100</number>
    </property>
   </widget>
   <widget class="QSpinBox" name="timeSelect">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>180</y>
      <width>71</width>
      <height>30</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>120</number>
    </property>
   </widget>
   <widget class="QLabel" name="timeText">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>140</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="statusTip">
     <string/>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Time (Minutes):</string>
    </property>
   </widget>
   <widget class="QLabel" name="loadText_2">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>111</width>
      <height>50</height>
     </rect>
    </property>
    <property name="whatsThis">
     <string/>
    </property>
    <property name="text">
     <string>Load:</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
